Alphabetised lists and added URLs to tools

patesson opened the pull request.

tpage99 commented:
Hey @patesson I'm looking into this PR. For some reason the Netlify preview builds are failing and may be an issue with MDX though I'm not really certain as to why. 

Just trying to do a little research - are you able to get this to build locally with gatsby develop?

patesson commented:
This worked for me before but ending the process and restarting caused the pages to fail. I've fixed the linting errors that came up but the issue seemed to be caused by the Index.md list having links and headers on each item. Removing the link or the header fixed it.

I've removed the header markdown to get it to build as the links are more useful than the styling.

tpage99 commented:
thanks @patesson! learned something new...agreed the links are what's most important
